vivisect/qt/main.py

```diff
@@ -107,6 +107,10 @@ def __init__(self, vw):
         self.addHotKeyTarget('file:open', self._menuFileOpen)
         self.addHotKey('ctrl+s', 'file:save')
         self.addHotKeyTarget('file:save', self._menuFileSave)
+        self.addHotKey('ctrl+meta+S', 'file:saveas')
```

Comment:
Did you mean to have the `s` to be lowercase? (same thing for the `c` on line 112)

Reply:
i'm open to the discussion, but i intended for all three modifiers to be pressed (ctrl/meta/shift) for these.  
ideally, we could keep the "ctrl+S" for save-as (ctrl/shift/s) but it's currently part of the Make Structs shortcuts.  

do you think i should make the change now, since most people are probably still unaware of the Make Structs shortcuts?

Reply:
So just `S` is for making structs. At least in memory mode. No ctrl. So `ctrl+S` is still open actually. So the change is wide open to made afaict.

Reply:
```
        self.addHotKey('S', 'viv:make:struct')
        self.addHotKey('ctrl+S', 'viv:make:struct:again')
        self.addHotKey('ctrl+meta+S', 'viv:make:struct:multi')
```

Reply:
huh.  makes me wonder if we shouldn't make sTructs (so everything is based around the letter 't')

Reply:
eventually, we should probably make all of these user-configurable :)  but that is not *this* day.

Reply:
done.  make save-as Ctrl-Shift-S.  migrated Struct hotkeys to the letter 't'.  let me know if you disagree.